changes/feature-7135-host-display-name

```diff
@@ -1,2 +0,0 @@
-* Add display_name to hosts, which is either the computer-name or the hostname.
```

Comment:
This doesn't seem to be mentioned in the updated CHANGELOG.md

Reply:
Ah, great catch. Adding this in.